src/components/calcite-split-button/calcite-split-button.scss

```diff
@@ -39,6 +77,64 @@
   .split-button__divider-container {
     background-color: var(--calcite-button-light);
   }
+  &:host([appearance="outline"]) {
+    .split-button__divider {
+      background-color: var(--calcite-button-light-text);
+    }
+    .split-button__divider-container {
+      background-color: var(--calcite-ui-foreground-1);
+      border-color: var(--calcite-button-light);
+    }
+    &:host(:hover) .split-button__divider-container {
+      background-color: var(--calcite-button-light);
+    }
+    &:host([theme="dark"]) {
+      .split-button__divider {
+        background-color: var(--calcite-ui-text-1);
+      }
+      .split-button__divider-container {
+        background-color: var(--calcite-ui-foreground-1);
+        border-color: var(--calcite-ui-foreground-1);
+      }
+    }
+  }
+  &:host([appearance="clear"]) {
+    .split-button__divider-container {
+      border-color: var(--calcite-button-light);
+    }
+    .split-button__divider,
+    &:host(:hover) .split-button__divider-container {
+      background-color: var(--calcite-button-light);
+    }
+    &:host([theme="dark"]) {
+      .split-button__divider,
+      &:host(:hover) .split-button__divider-container {
+        background-color: var(--calcite-button-dark);
+      }
+      .split-button__divider-container {
+        background-color: transparent;
+        border-color: var(--calcite-button-dark);
+      }
+    }
+  }
+  &:host([theme="dark"]) {
+    .split-button__divider-container {
+      background-color: var(--calcite-button-dark);
```

Comment:
The CSS vars in the top of this file could be replaced with theme-aliased CSS vars from calcite-colors, which could cut down on some of these dark specific rules: https://github.com/Esri/calcite-colors/blob/731942192f939993ceade97cffe2649724b56fd5/dist/colors.scss#L506